src/libcharon/plugins/windows_dns/windows_dns_handler.c

```diff
@@ -212,7 +212,7 @@ static int GetTapAdapterInfo( IWbemServices *pSvc, int *iface_idx, wchar_t *dns1
 	{
 		// Allocate our strings
 		language = SysAllocString( L"WQL" );
-		query = SysAllocString( L"SELECT * FROM Win32_NetworkAdapterConfiguration where Description LIKE 'Sophos TAP Adapter'" );
+		query = SysAllocString( L"SELECT * FROM Win32_NetworkAdapterConfiguration where Description LIKE 'Sophos TAP Adapter%'" );
```

Comment:
I am assuming that we expect either "Sophos TAP Adapter" or "Sophos TAP Adapter1", "Sophos TAP Adapter2"?  If this is correct, can we tighten the query like below?  Basically checking for prefix string "Sophos TAP Adapter" and then checking either empty or single digit number?

`where Description LIKE 'Sophos TAP Adapter%' AND (Description LIKE '%[0-9]' OR Description NOT LIKE '%[^0-9]')`